### PR TITLE
feat: add image support to billboard

### PR DESCRIPTION
**Pull request by cisel**
# Description
Related ticket: https://rvu.atlassian.net/secure/RapidBoard.jspa?rapidView=80&projectKey=MONEY&modal=detail&selectedIssue=MONEY-103

Screenshot 2021-07-30 at 12 54 24
Screenshot 2021-07-30 at 12 54 51
Screenshot 2021-07-30 at 15 27 06
Screenshot 2021-07-30 at 15 27 17
Screenshot 2021-07-30 at 15 23 10
Screenshot 2021-07-30 at 15 22 40

# Checklist

Pull request contains:

- [ ] A new component
- [x] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [x] Includes theme changes for both Uswitch and money
- [x] Work has been tested in multiple browsers
- [x] PR description includes description of change
- [x] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

File: src/compounds/billboard/src/stories.tsx [hunk 1]
```diff
@@ -118,6 +118,11 @@ const fullWidthSlot = (
   </Row>
 )
 
+const bgImage: string = text(
```
Comment (erkinun) [inline src/compounds/billboard/src/stories.tsx:121]: why dont we read a local static file?

**Review by erkinun — DISMISSED** [review]
LGTM but it'd still be helpful to see a screenshot without the image to make sure we persist the old functionality correct

**Comment by cisel** [comment]
I've updated the description with the screenshots @erkinun

File: src/compounds/billboard/src/stories.tsx (hunk 1 above)
Comment (cisel) [reply src/compounds/billboard/src/stories.tsx:121]: We can also use the static file. With this, we can change the source url on storybook without changing the code and see how different images look like. @erkinun

**Review by erkinun — APPROVED** [review]

**Pull request merged**